pyproject.toml

```diff
@@ -109,7 +109,7 @@ gpu = [ "cupy" ]
 cu12 = [ "cupy-cuda12x" ]
 cu11 = [ "cupy-cuda11x" ]
 # requests and aiohttp needed for zarr remote data
-lazy = [ "xarray>=2025.04.0", "aiohttp", "requests", "anndata[dask]" ]
+lazy = [ "xarray>=2025.04.0,<2025.06.0", "aiohttp", "requests", "anndata[dask]" ]
```

Comment:
Please always add a comment next to upper bounds that is sufficient to explain what needs to happen to remove the bound. An issue link is usually sufficient.

Reply:
Ah, can be removed now, things move quick: see #2008